presto-docs/src/main/sphinx/release/release-0.278.rst

```diff
@@ -0,0 +1,69 @@
+=============
+Release 0.278
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix :func:`ROUND` to prevent returning incorrect results due to integer / double overflows.
+* Fix the compilation error when aggregation has order by clause and the input is a function.
+* Optimize ``IF(predicate, AGG(x))`` to aggregation with mask in plan level. This is controlled by system property ``optimize_conditional_aggregation_enabled`` and defaults to false.
+* Add new security API ``selectAuthorizedIdentity`` and new configuration property ``permissions.authorized-identity-selection-enable`` to enable ``selectAuthorizedIdentity``. ``selectAuthorizedIdentity`` prevents potential security loopholes, e.g., reading illegal data from a fake username.
+* Add memory limit check for HashBuilderOperator during memory revoke.
+* Add null masking for the Parquet decryption feature. When this feature is enabled and the user is denied access encrypted column, the columns will be removed in the requested schema sent to Parquet. Then it is filled out with 'NULL'  when the result is returned.
+* Add optimization for :func:`approx_percentile` functions evaluation. Multiple :func:`approx_percentile` functions on the same field will be combined into one :func:`approx_percentile` function which takes an array of percentile as arguments. The optimization is controlled by session property ``optimize_multiple_approx_percentile_on_same_field`` which is true by default.
+* Add optimization for outer join by add randomized value for NULL join keys to avoid skew in NULL. This optimization is turned off by default and can be turned on by setting ``optimizer.randomize-outer-join-null-key`` to true.
+* Add retry with increased partition count if query fails due to low partition count. This can be enabled with the ``spark_hash_partition_count_scaling_factor_on_out_of_memory`` and ``spark_retry_on_out_of_memory_higher_hash_partition_count_enabled`` session properties.
+* Add function :func:`map_subset`. This function takes a map and an array of keys and returns a map with entries from the input map with keys contained in the array supplied.
+* Upgrade Apache Iceberg from 0.14.0 to 0.14.1.
+* Upgrade Java Topology Suite (jts) library version to 1.19.0.
```

Comment:
Is this user facing? If not, remove?

Reply:
Are you  saying the Java Topology Suite, I think it's user facing, as we use it in a lot of our geo functionalities?